test/util/test_ascii_art.py

```diff
@@ -95,3 +95,32 @@ def test_spot_counts_per_image_plot(dials_regression):
 1 image 10"""
     output = "\n".join(line.rstrip() for line in output.splitlines())
     assert output == expected_output
+
```

Comment:
Is it worth having this test on e.g. 1...9 images of centroid test data to make sure that it works _right_ for all the cases?

Reply:
The bug here was that the behaviour was inconsistent between even- and odd-length imagesets. We already had a test for output of even-length imagesets (1..10 above). I have added a test for an odd-length imageset below (1..3). We can change that to 1..9 if you think that would be better, but I don't think it would make any difference to the outcome.